strings/phobia.json

```diff
@@ -263,6 +263,38 @@
 		"wt550"
 	],
 
+	"heresy": [
+		"eldritch",
+		"ritual",
+		"rune",
+		"focus",
+		"codex cicatrix",
+		"heretic",
+		"mansus",
+		"rust",
+		"flesh",
+		"ash",
+		"void",
+		"blade",
+		"cosmic",
+		"space",
+		"star",
+		"armsy",
+		"worm",
+		"flesh snek",
+		"raw prophet",
+		"stalker",
+		"fire shark",
+		"ghoul",
+		"pierced reality",
+		"reality pierce",
+		"reality crack",
+		"maid in the mirror",
+		"hand",
+		"arm",
+		"void storm"
```

Comment:
"void" without "storm" is already in this same list.

Reply:
It's tangential but does the term "hand" also triggers with "hands"?